### PR TITLE
Stop posting PR results to pulls.web-platform-tests.org

### PR DESCRIPTION
**Pull request by foolip**
Resolves https://github.com/w3c/web-platform-tests/issues/10923.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/11060)

### CODE REVIEW

**Review by jugglinmike — APPROVED** [review]
I pushed up a dummy commit to demonstrate that the stability checker continues to operate as expected.

File: tools/ci/check_stability.py [hunk 1]
```diff
@@ -281,8 +231,6 @@ def run(venv, wpt_args, **kwargs):
 
     setup_logging()
 
-    browser_name = wpt_args.product.split(":")[0]
```
[the remark this answers is not in the document]
Comment (gsnedders) [reply tools/ci/check_stability.py]: We've had that turned off all the way back to 624c87efeffc7f53da0f2da336e9281b684b63a1. The initial whitelist was essentially just everything we failed at that point.

**Comment by foolip** [comment]
Thanks for checking that it actually still works @jugglinmike, an important consideration indeed :)

**Comment by jugglinmike** [comment]
Debugging through TravisCI is the most exciting part of my job!

File: tools/ci/check_stability.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jugglinmike) [reply tools/ci/check_stability.py]: Thanks for the context! What do you think about enabling that feature? https://github.com/w3c/web-platform-tests/pull/11097

**Pull request merged**